armi/cases/suiteBuilder.py

```diff
@@ -323,6 +323,10 @@ class LatinHyperCubeSuiteBuilder(SuiteBuilder):
     quasi-randomly into nonoverlapping partitions. It is recommended to use a surrogate
     model with the sampled data to get the full benefit.
 
+    .. impl:: A generic mechanism to allow users to modify user inputs in cases
+        :id: I_ARMI_CASE_MOD2
+        :implements: R_ARMI_CASE_MOD
```

Comment:
I don't think this belongs here.

You have it on the `inputModifier` class, and I think THAT is the important one.

Reply:
I think we can remove this one.  You have it covered on inputModifier.